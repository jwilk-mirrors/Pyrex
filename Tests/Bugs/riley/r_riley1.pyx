cdef int i
j = 0
i = 1
l = ['zero', 'one']
d = {0: 'two', 1: 'three'}
print l[j]
print l[i]
print d[j]
print d[i]
l[j] = 'Zero'
d[j] = 'Two'
print l[j]
print l[i]
print d[j]
print d[i]
l[i] = 'ONE'
d[i] = 'THREE'
print l[j]
print l[i]
print d[j]
print d[i]
