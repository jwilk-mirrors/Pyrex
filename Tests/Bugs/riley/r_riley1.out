zero
one
two
three
Zero
one
Two
three
Zero
ONE
Two
THREE
